man/mmrm_tmb_methods.Rd

```diff
@@ -155,7 +155,8 @@ object <- fit_mmrm(formula, fev_data, weights = rep(1, nrow(fev_data)))
 coef(object)
 # Fitted values:
 fitted(object)
-predict(object, newdata = fev_data)
+fit <- mmrm(FEV1 ~ RACE + SEX + ARMCD * AVISIT + us(AVISIT | USUBJID), data = fev_data)
```

Comment:
Correct? Did it not work with object?

Reply:
my bad. it should have worked. I did not see the object definition on the top.